yorkie-team/yorkie-js-sdk
# Fix quill paragraph style errors

#### What does this PR do?

Fix quill paragraph style errors

#### How should this be manually tested?

1. Input Texts
2. Apply paragraph attributes (e.g. h1, unordered list, etc)
3. Copy and paste the added string
4. Check for errors through the console

#### Any background context you want to provide?


#### What are the relevant tickets?

Fixes #58 

### Checklist
